pkg/apiserver/controller/fs/fs.go

```diff
@@ -179,22 +180,36 @@ func DeletePvPvc(fsID string) error {
 		return fmt.Errorf("list clusters failed")
 	}
 	for _, cluster := range clusters {
-		switch cluster.ClusterType {
-		case schema.KubernetesType:
-			runtimeSvc, err := runtime.GetOrCreateRuntime(cluster)
+		if cluster.ClusterType != schema.KubernetesType {
+			log.Debugf("cluster[%s] type: %s, no need to delete pv pvc", cluster.Name, cluster.ClusterType)
+			continue
+		}
+		runtimeSvc, err := runtime.GetOrCreateRuntime(cluster)
+		if err != nil {
+			return err
+		}
+		k8sRuntime := runtimeSvc.(*runtime.KubeRuntime)
+		namespaces := cluster.NamespaceList
+		if len(namespaces) == 0 { // cluster has no namespace restrictions. iterate all namespaces
+			nsList, err := k8sRuntime.ListNamespaces(metav1.ListOptions{})
 			if err != nil {
```

Comment:
log

Reply:
no need for log here, as log is printed where this func is invoked.